packages/cosmic-swingset/src/launch-chain.js

```diff
@@ -1458,7 +1458,22 @@ export async function launchAndShareInternals({
  */
 export async function launch(options) {
   const launchResult = await launchAndShareInternals(options);
-  return attenuate(launchResult, {
+  // TODO This really is a deep pick rather than a POLA abstraction like
+  // `attenuateOne`.
+  // Neither `launchResult` nor this attenuation of are passable. Nor could they
+  // be made passable since they mix methods and data properties. At the JS
+  // object level, as supported by HardenedJS itself, this could be made
+  // defensive. But
+  // - This `attenuate` abstraction seems to not even harden the attenuator
+  //   it returns, meaning that even at that low level it is already
+  //   non-defensive.
+  // - We are moving to a model where we treat only exo boundaries as
+  //   defensible, since exos bundle in interface guards, providing sound
+  //   declarative type-like input (and output) validation. This still leaves
+  //   remaining input validation up to the guarded method. But experience shows
+  //   that it is usually too hard to do all input validation without
+  //   guards or patterns. (That's why we created them.)
+  return deepPick(launchResult, {
```

Comment:
Comment seems a little out-of-date.  Maybe something like:
```suggestion
  // CAVEAT: `attenuateOne` provides a defensive POLA abstraction and should
  // be used instead of `deepPick`
  // Neither `launchResult` nor this `deepPick` of it are passable. Nor could they
  // be made passable since they mix methods and data properties. At the JS
  // object level, as supported by HardenedJS itself, this could be made
  // defensive. But
  // - The `deepPick` abstraction seems to not even harden the object
  //   it returns, meaning that even at that low level it is already
  //   non-defensive.
  // - We are moving to a model where we treat only exo boundaries as
  //   defensible, since exos bundle in interface guards, providing sound
  //   declarative type-like input (and output) validation. This still leaves
  //   remaining input validation up to the guarded method. But experience shows
  //   that it is usually too hard to do all input validation without
  //   guards or patterns. (That's why we created them.)
  return deepPick(launchResult, {
```

Reply:
Done, thanks!